src/command-parser/parserConfig.ts

```diff
@@ -130,6 +130,10 @@ const opCertSigningArgs = {
   },
 }
 
+// If you want to define a group of mutually exclusive CLI arguments (eg. see address.show below),
+// bundle these arguments under a key prefixed with '_mutually-exclusive-group'. Furthermore, if you
+// want argparse to ensure that one of the arguments is present, add '-required' suffix.
+
```

Comment:
This should mention that you can have several such groups and that they receive a name. (What's the purpose of the name, e.g. "payment"? Can it be left out?)

Reply:
Good point, explanation added in https://github.com/vacuumlabs/cardano-hw-cli/pull/109/commits/955aadbe22fe208731b7671a099156c7cf91eb1e.